index.html

```diff
@@ -49,5 +49,167 @@ <h2>I'am a Sotfware Developer</h2>
           </div>
         </section>
     </main>
+    <div class="main-body">  
+      <section class="main-row">
+        <h1>My Recent work</h1>
+        <div class="h-line"><hr></div>
+        <div class="story-section">
+          <img src="images/st-img.png" alt="st-imag">
+          <div class="h1-tag"><h1>Multi-Post Stories</h1></div>
```

Comment:
- [ ] Let's apply my last comment to each one of the `h1` tags that we see in our HTML.